test.html

```diff
@@ -164,12 +281,72 @@ <h2>Test drive</h2>
 		});
 }
 
+/**
+ * Loads a plugin including it css file
+ *
+ * @param {string} pluginName the plugin to load
+ * @type {Promise} the promise which resolves as soon as plugin is loaded
+ */
+function loadPlugin (pluginName)
+{
+	var plugin = components.plugins[pluginName];
+	if (!plugin){
+		return;
```

Comment:
I _think_ this should return `Promise.resolve()` to make the return value consistent?

Reply:
Agree, updating